setup.py

```diff
@@ -19,8 +22,8 @@
 else:
     INSTALL_REQUIRES = [
         'esmpy>=8.0.0',
-        'xarray <= 0.16.0',
-        'numpy >=1.16',
+        'xarray<=0.16.0',
```

Comment:
```suggestion
        'xarray!=0.16.1',
```

shouldn't this be enough?